src/js/main.js

```diff
@@ -218,4 +220,26 @@ function navigateResultPages(
   }
 }
 
+theme.modeToggle(theme.root, theme.themeToggle);
+
+function removeMode(searchInput) {
+  if (searchInput) {
+    if (localStorage.getItem("mode")) {
+      localStorage.removeItem("mode");
+    }
+  }
+}
+
+removeMode(searchInput);
```

Comment:
What is the purpose of this?

Reply:
prvi if: brez tega se požene funkcija tudi, ko se odpre subpage (ker vleče celo skripto, namesto samo exportov)
drugi if: brez tega ob refreshu domače strani ostane mode: dark v localStorage, čeprav refreshana stran po defaultu ni v dark. Posledica je, da če potem sprožiš iskanje in odpreš subpage enega od zadetkov, se ta odpre v dark (namesto v tisti barvni shemi, v kateri je domača stran)

Reply:
ne razumem zakaj sploh brišeš `mode` iz local storaga? a ni fajn da ostane shranjen?

Reply:
saj hoces, da bo user imel tist mode, ki si ga je izbral, vsakič, ko bo obiskal stran

Reply:
ja lahko je kombinacija obojga.
kaj misls da je bol anoying, da moras vsakic ponovno nastavit dark theme, al da si stran zapomni tvojo nastavitev?

Reply:
aha ok. 
po občutku bi rekel, da v večini primerov prvo.
bom popravil.